app/src/views/DrefFinalReportExport/index.tsx

```diff
@@ -550,12 +549,14 @@ export function Component() {
                         valueType="boolean"
                         strongLabel
                     />
-                    <BlockTextOutput
-                        label={strings.nsInterventionDescriptionLabel}
-                        value={drefResponse?.national_society_conducted_description}
-                        valueType="text"
-                        strongLabel
-                    />
+                    {drefResponse?.has_national_society_conducted === true && (
```

Comment:
```suggestion
                    {!!drefResponse?.has_national_society_conducted && (
```

Reply:
@AdityaKhatri We are actually check if the user has answered `Yes`